packages/xo-web/src/common/usage/index.js

```diff
@@ -2,30 +2,39 @@ import _ from 'intl'
 import classNames from 'classnames'
 import PropTypes from 'prop-types'
 import React, { cloneElement } from 'react'
-import { compact, sum } from 'lodash'
+import { compact, sumBy } from 'lodash'
 
 import Tooltip from '../tooltip'
 
-const Usage = ({ total, children }) => {
+const Usage = ({ total, children, link }) => {
   const limit = total / 400
-  const othersValues = compact(
+  const othersProps = compact(
     React.Children.map(children, child => {
       const { value } = child.props
-      return value < limit && value
+      return value < limit && child.props
     })
   )
-  const othersTotal = sum(othersValues)
-  const nOthers = othersValues.length
+  const othersTotal = sumBy(othersProps, 'value')
+  const nOthers = othersProps.length
   return (
     <span className='usage'>
       {nOthers > 1 ? (
         <span>
           {React.Children.map(
             children,
             (child, index) =>
-              child.props.value > limit && cloneElement(child, { total })
+              child.props.value > limit &&
+              cloneElement(child, {
+                href: typeof link === 'function' ? link(child.props.id) : link,
```

Comment:
Should it override the `href` prop if it was already set manually? Or should should we show a warning in the console if there's both a `link` prop on `Usage` *and* an `href` prop on one of the `Element`s, maybe?

Reply:
If `href` is already passed, we keep it
```js
href: 
   child.props.href === undefined 
   ? typeof link === 'function'  ? link(child.props.id) : link 
   : child.props.href 
```
What do you think?